dts/include/coronet/a12-cpu.dtsi

```diff
@@ -43,11 +43,51 @@
 				};
 			};
 
-			mv6_port(0, "ethX2" , &aphy0, "gmii");
+			agate_port0: port@0 {
+				phy-handle = <&aphy0>;
+				reg = <0>;
+				label = "ethX2";
+				local-mac-address = [00 00 00 00 00 00];
+				phy-mode = "gmii";
+			};
+
+			agate_port1: port@1 {
+				phy-handle = <&aphy1>;
+				reg = <1>;
+				label = "ethX12";
+				local-mac-address = [00 00 00 00 00 00];
+				phy-mode = "gmii";
+			};
+
+			agate_port2: port@2 {
+				phy-handle = <&aphy2>;
+				reg = <2>;
+				label = "ethX8";
+				local-mac-address = [00 00 00 00 00 00];
+				phy-mode = "gmii";
+			};
+
+			agate_port3: port@3 {
+				phy-handle = <&aphy3>;
+				reg = <3>;
+				label = "ethX4";
+				local-mac-address = [00 00 00 00 00 00];
+				phy-mode = "gmii";
+			};
+
+			agate_port4: port@4 {
+				phy-handle = <&aphy4>;
+				reg = <4>;
+				label = "ethX3";
+				local-mac-address = [00 00 00 00 00 00];
+				phy-mode = "gmii";
+			};
+
+			/* mv6_port(0, "ethX2" , &aphy0, "gmii");
 			mv6_port(1, "ethX12", &aphy1, "gmii");
 			mv6_port(2, "ethX8" , &aphy2, "gmii");
 			mv6_port(3, "ethX4" , &aphy3, "gmii");
-			mv6_port(4, "ethX3" , &aphy4, "gmii");
+			mv6_port(4, "ethX3" , &aphy4, "gmii"); */
```

Comment:
FYI: When replacing a chunk of code you do not need to keep the old one commented out. We have version control systems for this exact purpose.